packages/opentelemetry/src/types.ts

```diff
@@ -0,0 +1,36 @@
+import { Metric } from '@theniledev/js';
+import { ExportResult } from '@opentelemetry/core';
+
+/**
+ * Options for Jaeger configuration
+ */
+export interface ExporterConfig {
+  /**
+   * The base path of the Nile host endpoint, i.e. https://prod.thenile.dev.
+   */
+  basePath?: string;
+  /**
+   * The Nile instance id for the calling context.
+   */
+  instanceId: string;
```

Comment:
This would probably need to be extracted some other way - right? i.e., Whatever is using this would probably be dealing with more than one instance?